generators/bitrise/README.md

```diff
@@ -0,0 +1,38 @@
+# Bitrise setup for continuous deployment
+
+:warning: ***Having set up Fastlane with `yo rn-toolbox:fastlane` is a requirement***
```

Comment:
Je te recommande un petit
```
end: function() {
    this.config.set('fastlane', true);
    this.config.save();
  }
```

Dans la conf fastlane et un check au lancement de ce script avec un throw si `!this.config.get('fastlane');`